.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: 10.2.0
```

Comment:
We should move to - 10.3.0
it is supported by circle ci

Reply:
done